packages/mgt-chat/src/utils/onRenderMessage.tsx

```diff
@@ -0,0 +1,153 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+
+import { MessageProps, MessageRenderer } from '@azure/communication-react';
+import * as AdaptiveCards from 'adaptivecards';
```

Comment:
Let's just import the things we want here:
```suggestion
import { Action, AdaptiveCard, IMarkdownProcessingResult } from 'adaptivecards';
```